x-pack/plugins/security_solution/public/management/pages/event_filters/state/index.ts

```diff
@@ -19,6 +19,13 @@ export interface EventFiltersPageLocation {
   filter: string;
 }
 
+export interface EventFiltersListPageData {
```

Comment:
Should we move this to types.ts?

Reply:
yes, all of this should be moved to `types.ts`. I will do it in next PR